modules/odo-core-concepts.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-core-concepts_{context}"]
+
+= odo core concepts
+
+`{odo-title}` abstracts Kubernetes concepts into a developer friendly terminology:
+
+Application::
+An _application_ in `odo` is a classic application, developed with a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that is used to perform a particular task.
```

Comment:
```suggestion
An _application_ in `odo` is a classic application, developed with a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], to perform a particular task.
```

Reply:
```suggestion
A classic application that has been decomposed into microservices and reployed as a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native application].
```
My best guess is that this is what we mean to say...

Reply:
I believe the intention of the word "classic" here is to convey the sense of "typical" or "normal", rather that incorporating the idea of monoliths. I will use "typical" and let the SME decide.